src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestGHEventSubscriber.java

```diff
@@ -59,11 +59,13 @@
 
     @Override
     protected boolean isApplicable(@Nullable Job<?, ?> project) {
-        if (project.getParent() instanceof SCMSourceOwner) {
-            SCMSourceOwner owner = (SCMSourceOwner) project.getParent();
-            for (SCMSource source : owner.getSCMSources()) {
-                if (source instanceof GitHubSCMSource) {
-                    return true;
+        if (project != null) {
+            if (project.getParent() instanceof SCMSourceOwner) {
+                SCMSourceOwner owner = (SCMSourceOwner) project.getParent();
+                for (SCMSource source : owner.getSCMSources()) {
+                    if (source instanceof GitHubSCMSource) {
+                        return true;
+                    }
```

Comment:
Yes. I need it. This new parent pom includes [Findbugs 3.0.3](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L328) and this version detects an issue.